src/style.css

```
@@ -1,3 +1,62 @@
-body {
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: #00ACDC;
+}
+
+body {
+  font-family: Roboto, sans-serif;
+}
+
+header {
```

Need a use a classes selector
```suggestion
.header {
```